From 60f127241193694fa369adef134bdf3dd4bb9e8d Mon Sep 17 00:00:00 2001
From: Adam Farden <adam@farden.cz>
Date: Thu, 9 Aug 2018 09:26:47 +0200
Subject: [PATCH] Add msm8976 tasha sound card detection to msm8916 HAL

msm8976 is a variant of msm8952, both use the msm8916 HAL

Test: manual - build ok. Sound ok

Change-Id: I9ac11e781cf627fa5efe586c96e48bfd04f32485
Signed-off-by: Adam Farden <adam@farden.cz>
---
 hal/msm8916/hw_info.c | 8 +++++---
 1 file changed, 5 insertions(+), 3 deletions(-)

diff --git a/hal/msm8916/hw_info.c b/hal/msm8916/hw_info.c
index 0afb510..095d052 100644
--- a/hal/msm8916/hw_info.c
+++ b/hal/msm8916/hw_info.c
@@ -54,8 +54,10 @@ static void update_hardware_info_8x16(struct hardware_info *hw_info, const char
         strlcpy(hw_info->name, "msm8952", sizeof(hw_info->name));
     }  else if (!strcmp(snd_card_name, "msm8952-l9300-snd-card")) {
         strlcpy(hw_info->name, "msm8952", sizeof(hw_info->name));
+    } else if (!strcmp(snd_card_name, "msm8976-tasha-snd-card")) {
+        strlcpy(hw_info->name, "msm8976", sizeof(hw_info->name));
     } else {
-        ALOGW("%s: Not an  8x16/8909/8952 device", __func__);
+        ALOGW("%s: Not an 8x16/8909/8952/8976 device", __func__);
     }
 }
 
@@ -69,8 +71,8 @@ void *hw_info_init(const char *snd_card_name)
         return NULL;
     }
 
-    if (strstr(snd_card_name, "msm8x16") || strstr(snd_card_name, "msm8909")
-        || strstr(snd_card_name, "msm8952") ||
+    if (strstr(snd_card_name, "msm8x16") || strstr(snd_card_name, "msm8909") ||
+        strstr(snd_card_name, "msm8952") || strstr(snd_card_name, "msm8976") ||
         strstr(snd_card_name, "msm-bg-snd-card")) {
         ALOGV("8x16 - variant soundcard");
 
-- 
2.13.0

